haros/haros.py

```diff
@@ -459,11 +468,11 @@ class HarosAnalyseRunner(HarosCommonExporter):
                   + os.environ.get("ROS_DISTRO", "kinetic")
                   + "/distribution.yaml")
 
-    def __init__(self, haros_dir, project_file, data_dir, whitelist, blacklist,
+    def __init__(self, haros_dir, config_dir, project_file, data_dir, whitelist, blacklist,
```

Comment:
Same as above. Replace `config_dir` with `config_path`, since you are expecting a full path to the config file.